src/theme/ui_styles.rs

```diff
@@ -89,10 +89,12 @@ pub struct Size {
 #[rustfmt::skip]
 #[derive(Clone, Copy, Debug, Default, PartialEq)]
 pub struct Users {
-    pub user_you: Style,           // uu
-    pub user_someone_else: Style,  // un
-    pub group_yours: Style,        // gu
-    pub group_not_yours: Style,    // gn
+    pub user_you: Style,                    // uu
+    pub user_someone_else_not_root: Style,  // un
```

Comment:
This is for a user that isn't you, and isn't root right?

If so isn't there:
- you
- root
- other

Because in that case, perhaps we could go with the shorter:
```suggestion
    pub user_other: Style,  // un
```